tutorials/howtos/trim-index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "How can I remove pieces from an index listing?"
+layout: tutorial
+---
+
+In Apostrophe, an index page (or paginated pages) is a listing of "pieces" on a page or across paginated pages powered by the `apostrophe-pieces-pages` module or a module that extends `apostrophe-pieces-pages`. A list of blog posts and staff profiles page are both possible examples.
```

Comment:
This is still a weird paragraph somehow. How about replacing it with:

In Apostrophe, you'll often display a nice paginated list of pieces on a page, using a module that extends apostrophe-pieces-pages. That includes modules like apostrophe-blog-pages that extend apostrophe-pieces-pages. We'll refer to these special pages as "index pages."

But, I'm calling it good - thanks for writing this up, feel free to take this last suggestion or just merge!